File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/utilClass/varConfigurations/VarConfig.java
package org.firstinspires.ftc.teamcode.utilClass.varConfigurations;

import com.acmerobotics.dashboard.config.Config;

// these variables can be changed from FTC dashboard
@Config
public class VarConfig {
    public static double pitch = 400.0;
    public static double extend = 400.0;
    public static boolean loopSaver = false;
    public static boolean autoRotateClaw = true;
    public static double slowMult = 3;

    public static boolean relocalize = true;

    public static double fieldRadius = 10.0;
    public static double minRectArea = 300;
    public static double correctionSpeedAvoid = 0.01;
    public static double robotRadiusAvoidance = 8.0;
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/UtilClass/varConfigurations/varConfig.kt
package org.firstinspires.ftc.teamcode.UtilClass.varConfigurations

import com.acmerobotics.dashboard.config.Config

// these variables can be changed from FTC dashboard
@Config
object varConfig {
    @JvmField
    var delay = 1

    @JvmField
    var slowMult = 3

    @JvmField
    var minConfidence = 0.5

    @JvmField
    var minRectArea = 300.0

    @JvmField
    var useFileWriter = true

    @JvmField
    var multipleDrivers = true

    @JvmField
    var usePIDF = true

    @JvmField
    var usingAvoidance = true

    @JvmField
    var fieldRadius = 8.0

    @JvmField
    var correctionSpeedAvoid = 0.02

    @JvmField
    var robotRadiusAvoidance: Double = 8.0
}
